### DISCUSSION TITLE
Create a discord community for the organization

### DISCUSSION THREAD

**Discussion by vedantmgoyal9** [App-Choreography/OurContributors#40] [Team Posts]
GitHub is meant for professional communication, sharing ideas, reporting bugs, and requesting features. Since this is an open community organization, where people would definitely like to know each other, talk & hang out, share jokes & memes, etc... I would suggest to setup a discord server where the organization members can join, and get to know each other. Also, this can help in growth of our organization since the discord invites can be shared in other discord servers, and then they can also join our GitHub organization, if they would like to.

Pinging whole community (@App-Choreography/community) on this one to get feedback of existing members!

**Comment by 0xMRTT** [comment]
Bringing to https://matrix.org

**Comment by vedantmgoyal9** [comment]
I have no issues with creating a chat space on the matrix as well, but I've seen many popular channels of PowerShell, Neovim, and even Python remain inactive. Also, there aren't any "good" bots and customizations that we can do on that platform. The point is - why not adapt a platform which is more widely used by the communities?

**Comment by 0xMRTT** [comment]
We can make a discord server which is bridged to a matrix space

**Comment by trudever** [comment]
right

**Comment by vedantmgoyal9** [comment]
> We can make a discord server which is bridged to a matrix space

Yup, that can be done.

**Comment by vedantmgoyal9** [comment]
@CodingSpecies what are your thoughts?
